key.ui/src/main/java/de/uka/ilkd/key/gui/docking/DockingLayout.java

```diff
@@ -245,7 +146,7 @@ final class SaveLayoutAction extends MainWindowAction {
     private static final long serialVersionUID = -2646217961498111734L;
     private final String layoutName;
 
-    public SaveLayoutAction(MainWindow mainWindow, String name, Integer key) {
+    SaveLayoutAction(MainWindow mainWindow, String name, Integer key) {
         super(mainWindow);
         this.layoutName = name;
         setName("Save as " + name);
```

Comment:
```suggestion
        setName("Save " + name);
```
"Save as" does not really make sense. It should be "Save to", but to have a cleaner menu, I would just remove the preposition.